src/view/com/util/TimeElapsed.tsx

```diff
@@ -15,9 +13,11 @@ export function TimeElapsed({
   const tick = useTickEveryMinute()
   const [timeElapsed, setTimeAgo] = React.useState(() => ago(timestamp))
 
-  React.useEffect(() => {
+  const [prevTick, setPrevTick] = React.useState(tick)
+  if (prevTick !== tick) {
+    setPrevTick(tick)
```

Comment:
isn't that (bad setState in render)? 

I mean this error:
> `Warning: Cannot update a component (`App`) while rendering a different component (`Watever`). To locate the bad setState() call inside `Testing`, follow the stack trace as described in https://reactjs.org/link/setstate-in-render

Reply:
That error is about updating a _different_ component. Updating the same component is ok if it's following this specific pattern: https://react.dev/reference/react/useState#storing-information-from-previous-renders